app/javascript/components/property-mapping-list/stores/propertyMappingListStore.js

```diff
@@ -44,18 +43,14 @@ export const defaultState = {
   propertiesInputValue: '',
 };
 
-export const propertyClassesForSpineTerm = (term) => {
-  const selectedDomains = uniq(
-    flatMap(term.alignments, (alignment) => alignment.selectedDomains || [])
-  );
-  const termClasses = term.property.domain || [];
-  return intersection(selectedDomains, termClasses);
-};
+export const propertyClassForSpineTerm = (term) => {
+  const { selectedDomain } = term.property;
+
+  if ((selectedDomain ?? '').startsWith('http')) {
```

Comment:
guess we need to add this checking to previous implementation otherwise [this requirement](https://github.com/t3-innovation-network/desm/issues/426#issuecomment-1992245460) will be lost.